api/net/Areas/Editor/Models/Content/FolderModel.cs

```diff
@@ -0,0 +1,51 @@
+using TNO.Core.Extensions;
+
+namespace TNO.API.Areas.Editor.Models.Content;
+
+/// <summary>
+/// FolderModel class, provides a model that represents an file or directory.
+/// </summary>
+public class FolderModel
```

Comment:
What is this model used for within this PR?

Reply:
I thought I had included this because I was getting compilation errors without it, but this seems not to be the case. I've removed the file.